code/common/finspace.q

```diff
@@ -5,6 +5,7 @@ enabled:@[value;`enabled;0b];                               //whether the applic
 database:@[value;`database;"database"];                     //name of the finspace database applicable to a certain RDB cluster - Not used if on prem
 
 hdbclusters:@[value;`hdbclusters;enlist `cluster];          //list of clusters to be reloaded during the rdb end of day (and possibly other uses)
+rdbready:@[value;`rdbready;0b];                             //whether or not the rdb is running and ready to take over at the next period- set to false by default
```

Comment:
this will be set in all processes, not just rdb processes - is that intended?

Reply:
yes for all rdbs/wdbs